elementary/monitor/api/totals_schema.py

```diff
@@ -0,0 +1,31 @@
+from pydantic import BaseModel
+
+
+class TotalsSchema(BaseModel):
```

Comment:
Was defined twice with exactly the same code (which caused wrong type references), so moved to a shared location